### DISCUSSION TITLE
SPUNTA FIRMA ORA DOCENTE

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#136] [Idee e proposte]
BUONASERA,SAREBBE POSSIBILE FAR MODO CHE IL DOCENTE SPUNTI IN QUALCHE PARTE PER EFFETTUARE UNA VERA E PROPRIA FIRMA PRIMA DELLA COMPILAZIONE DELLO STESSO REGISTRO?ALLEGO IMMAGINE RIPRESA DAL REGISTRO ELETTRONICO  ARGO 
https://user-images.githubusercontent.com/93823125/150651525-26c018ec-e79d-4bc8-bcb9-6d3da82c765c.png

**Comment by trinko** [comment]
No, non serve.
Il docente CLICCA sul pulsante per aggiungere/modificare la lezione, e questa procedura ha come esito di visualizzare la sua firma dell'ora.
